File: app/src/at-tamle/java/asiantech/internship/winter/ex28/Standardiz.java
package asiantech.internship.winter.ex28;
import java.util.Scanner;
import java.util.StringTokenizer;
public class Standardiz {
    public static String New(String str){
        String s,b;
        s= str.substring(0, 1);
        b= str.replaceFirst(s,s.toUpperCase());
        return (b);
    }
    private static String equal(String strInput){
        String b="";
        StringTokenizer strToken= new StringTokenizer(strInput," ,\t,\r");
        b+=""+New(strToken.nextToken()); while(strToken.hasMoreTokens()){
            b +=" "+New(strToken.nextToken());
        }
        return b;
    }
    public static void main(String[] args) {

        Scanner input= new Scanner(System.in);
        System.out.println("Nhap vao 1 xau: ");
        String strInput= input.nextLine();
        System.out.println("Xau duoc chuan hoa la: "+equal(strInput));
    }
}

File: app/src/at-tamle/java/asiantech/internship/summer/Kotlin/Standardiz.kt
package asiantech.internship.summer.Kotlin
import java.util.Scanner
import java.util.StringTokenizer
object Standardiz {
    fun New(str: String): String {
        val s: String
        val b: String
        s = str.substring(0, 1)
        b = str.replaceFirst(s.toRegex(), s.toUpperCase())
        return b
    }
    fun equal(strInput: String): String {
        var b = ""
        val strToken = StringTokenizer(strInput, " ,\t,\r")
        b += "" + New(strToken.nextToken())
        while (strToken.hasMoreTokens()) {
            b += " " + New(strToken.nextToken())
        }
        return b
    }

    @JvmStatic
    fun main(args: Array<String>) {

        val input = Scanner(System.`in`)
        println("Nhap vao 1 xau: ")
        val strInput = input.nextLine()
        println("Xau duoc chuan hoa la: " + equal(strInput))
    }
}
